Advanced-Recruit-Training/src/I_BatteryStateOfChargeService.h

```diff
@@ -13,4 +13,6 @@ class I_BatteryStateOfChargeService
     virtual QTime timeWhenChargedOrDepleted() const = 0;
 
     virtual void addData(const BatteryData& batteryData) = 0;
+//private:
+//    int counter=0;
```

Comment:
No need for commented code. I would just like to make an additional note as well, this is an interface class as denoted by the `I_`, there should never be any private members in an interface.